categories/software-engineering/rules-to-better-large-builds-in-visual-studionet.md

```diff
@@ -0,0 +1,13 @@
+---
+type: category
+title: SSW Rules to Better Large Builds in Visual Studio.NET
```

Comment:
@bradystroud @jaydenalchin

Can this category be archived?

Reply:
I don't know enough on the topic to confirm or deny @JackDevAU